packages/tinacms/src/auth/TinaCloudProvider.tsx

```diff
@@ -173,6 +174,14 @@ export const TinaCloudProvider = (
       }
     }
   }
+  /**
+   * Attaches a new instance of the TinaAdminApi to `cms.api.admin`
+   */
+  const setupAdmin = async () => {
+    if (!cms.api.admin) {
+      cms.registerApi('admin', new TinaAdminApi(cms))
+    }
+  }
```

Comment:
Register the `admin` API alongside the `tina` one for making calls to `TinaAdminApi` (specifically for `fetchCollections()` in the `Nav`).